servo-pca9685/src/lib.rs

```diff
@@ -0,0 +1,112 @@
+//! http://cache.nxp.com/documents/data_sheet/PCA9685.pdf?pspll=1
+
+extern crate tessel;
+
+use std::io;
+use std::thread;
+use std::time::Duration;
+use std::ops::Range;
+
+#[repr(u8)]
+#[allow(dead_code)]
+#[derive(Copy, Clone)]
+enum Command {
+    MODE1 = 0x0,
+    LED0_ON_L = 0x06,
+    LED0_ON_H = 0x07,
+    LED0_OFF_L = 0x08,
+    LED0_OFF_H = 0x09,
+    PRESCALE = 0xFE,
+}
+
+const MAX: u16 = 4096;
+//const MODE1 = 0x0;
+//const PRE_SCALE = 0xFE;
+
+const I2C_ID: u8 = 0x73;
+
+#[allow(dead_code)]
+pub struct ServoArray<'a> {
```

Comment:
Actually Servo array (a physical array of servos) but I don't like the name. Opening an issue